api-reference/10 UI Components/dxOverlay/1 Configuration/copyRootClassesToWrapper.md

```diff
@@ -5,7 +5,28 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Copies your custom CSS classes from the root element to the wrapper element. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property allows you to use CSS for wrapper element configuration. You can customize the shade, resize, and drag zones.
```

Comment:
```suggestion
This property allows you to use CSS for wrapper element configuration. You can resize, drag, and customize the shade of zones.
```

Reply:
Hello @dirk-pieterse 

Seems like my text is not clear enough. The point is the following: we have three zones - the drag zone, the resize zone and the shade zone. Users can customize each one of these zones.

Reply:
I misunderstood the initial meaning of your sentence. It didn't seem like different zones, rather different things you can do with a zone.